contracts/contracts/zapper/WOETHCCIPZapper.sol

```diff
@@ -146,4 +172,30 @@ contract WOETHCCIPZapper {
         // Return the message ID
         return messageId;
     }
+
+    function _estimateFee(uint256 amount, address receiver)
```

Comment:
This method is called estimateFee. Is this calculating the exact fee, or could it be wrong? If it's not exact, what happens when high or low?

Reply:
> returns guaranteed execution fee for the specified message delivery to the destination chain

If this is true from the chainlink docs, perhaps we should remove the "estimate" from the name.